### PR TITLE
feat: ID-1502: sample app examples for create seaport listing

### PR DESCRIPTION
**Pull request by scastleman-immutable**
# Summary
Additional examples for the SDK sample app for creating seaport listing message signings

### CODE REVIEW

File: packages/passport/sdk-sample-app/src/components/zkevm/EthSendTransactionExamples/SeaportCreateListingDefault.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+    return handleExampleSubmitted({
+      method: 'eth_signTypedData_v4',
+      params,
+    });
+  }, [handleExampleSubmitted, params]);
+
+  return (
+    <Accordion.Item eventKey="4">
+      <Accordion.Header>Seaport Create Listing (Generic)</Accordion.Header>
```
Comment (haydenfowler) [inline packages/passport/sdk-sample-app/src/components/zkevm/EthSendTransactionExamples/SeaportCreateListingDefault.tsx:53]: Can we rename this wording? I worry that people may associate `Generic` with some sort of Generic confirmation screen 😅 

```suggestion
      <Accordion.Header>Seaport Create Listing (Hardcoded example)</Accordion.Header>
```

File: packages/passport/sdk-sample-app/src/components/zkevm/EthSendTransactionExamples/SeaportCreateListing.tsx [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+            </Form.Label>
+            <Form.Control
+              required
+              value={tokenContractAddress}
+              isValid={transaction && !transactionError}
+              isInvalid={!!transactionError}
+              onChange={(e) => setTokenContractAddress(e.target.value)}
+            />
+          </Form.Group>
+          )}
```
Comment (haydenfowler) [inline packages/passport/sdk-sample-app/src/components/zkevm/EthSendTransactionExamples/SeaportCreateListing.tsx:196]: Nitpick: Can I suggest that we display the `Type` before the amount?

```suggestion
          <Form.Group className="mb-3">
            <Form.Label>
              Currency Type
            </Form.Label>
            <Form.Select
              required
              value={buyType}
              onChange={(e) => setBuyType(e.target.value as TokenType)}
            >
              <option value="NATIVE">Native</option>
              <option value="ERC20">ERC20</option>
            </Form.Select>
          </Form.Group>
          {buyType === 'ERC20' && (
          <Form.Group className="mb-3">
            <Form.Label>
              Currency Contract Address
            </Form.Label>
            <Form.Control
              required
              value={tokenContractAddress}
              isValid={transaction && !transactionError}
              isInvalid={!!transactionError}
              onChange={(e) => setTokenContractAddress(e.target.value)}
            />
          </Form.Group>
          )}
          <Form.Group className="mb-3">
            <Form.Label>
              Currency Amount
            </Form.Label>
            <Form.Control
              required
              type="text"
              value={buyAmount}
              isValid={transaction && !transactionError}
              isInvalid={!!transactionError}
              onChange={(e) => setBuyAmount(e.target.value)}
            />
          </Form.Group>
```

**Review by haydenfowler — COMMENTED** [review]
Nice 🙌 tested locally and everything works well!

File: packages/passport/sdk-sample-app/src/components/zkevm/EthSendTransactionExamples/SeaportCreateListing.tsx (hunk 2 above)
Comment (scastleman-immutable) [reply packages/passport/sdk-sample-app/src/components/zkevm/EthSendTransactionExamples/SeaportCreateListing.tsx:196]: image
It looks a little strange, but that might just be me. If you're all good with this we can post it though

Comment (haydenfowler) [reply packages/passport/sdk-sample-app/src/components/zkevm/EthSendTransactionExamples/SeaportCreateListing.tsx:196]: What's the concern there? Does renaming `Token Contract Address` to `Currency Contract Address` alleviate your concerns?

Comment (scastleman-immutable) [reply packages/passport/sdk-sample-app/src/components/zkevm/EthSendTransactionExamples/SeaportCreateListing.tsx:196]: Yeah I think that might make more sense, keeps the consistency with naming. I'll change it and send it

File: packages/passport/sdk-sample-app/src/components/zkevm/EthSendTransactionExamples/SeaportCreateListing.tsx [hunk 3]
```diff
@@ -85,6 +85,7 @@ function SeaportCreateListing({ disabled, handleExampleSubmitted }: RequestExamp
       );
 
       const payload = typedDataEncoder.getPayload(populated.domain, signAction!.message!.types, populated.value);
+      console.log(payload);
```
Comment (haydenfowler) [inline packages/passport/sdk-sample-app/src/components/zkevm/EthSendTransactionExamples/SeaportCreateListing.tsx]:
```suggestion
```

**Review by haydenfowler — APPROVED** [review]

**Review by imx-mikhala — APPROVED** [review]

**Pull request merged**